File: Java/6.2 - Behaviour driven development (BDD)/src/main/java/com/qa/core/Calculator.java
package com.qa.core;

public class Calculator {
	private int answer;

	public Calculator() {
		this.answer = 0;
	}

	public void addTwoNumber(int x, int y) {
		this.answer = x + y;
	}

	public int getresult() {
		return this.answer;
	}
}

File: C#/6.2 - BDD/QACalculator/Calculator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QACalculator
{
    public class Calculator
    {
        private int answer;

        public Calculator()
        {
            this.answer = 0;
        }

        public void AddTwoNumber(int x, int y)
        {
            this.answer = x + y;
        }

        public int GetResult()
        {
            return this.answer;
        }
    }
}
